products/amazon-cdk.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Amazon CDK
+category: framework
+iconSlug: amazonaws
+permalink: /amazon-cdk
+changelogTemplate: https://github.com/aws/aws-cdk/releases/tag/v__LATEST__
+releaseDateColumn: true
```

Comment:
This could do with some tags.

Reply:
Added some, let me know if you want me to add/remove tags

Reply:
Looked at other tags in the repo, and `amazon` seems the only relevant one. So removed the rest.